drivers/dma/dma_stm32.c

```diff
@@ -324,6 +324,10 @@ static int dma_stm32_configure(struct device *dev, u32_t id,
 		periph_addr_adj = config->head_block->dest_addr_adj;
 		break;
 	/* Direction has been asserted in dma_stm32_get_direction. */
+	default:
+		LOG_ERR("Channel direction error. %d",
```

Comment:
maybe you may want to get rid of the '.' here.

Reply:
changed to` LOG_ERR("Channel direction error (%d)."`

Reply:
ack